src/distortion.js

```diff
@@ -62,7 +124,8 @@ class Distortion extends Effect {
      */
     this.waveShaperNode = this.ac.createWaveShaper();
 
-    this.amount = curveAmount;
+    //this.amount = curveAmount;
```

Comment:
I think we can just delete this and line 164 instead of leaving them commented out